### ISSUE TITLE
AppImaged generates incorrect .desktop

### ISSUE THREAD

**Issue by TheLastProject** [Pext/Pext#105]
Probably due to the sed statement on pext.desktop in travis/build-appimage.sh.

```
[Desktop Entry]
Type=Application
Name=Pext
Comment=Python-based extendable tool
Categories=Utility;
Exec=/home/svos/Downloads/Pext-x86_64.AppImage usr/bin/pext
Icon=appimagekit_e92124c08636533f1af2b417e6e8f460_pext
StartupNotify=false
Terminal=false
TryExec=/home/svos/Downloads/Pext-x86_64.AppImage
X-AppImage-Comment=Generated by appimaged 0961d18
X-AppImage-Identifier=e92124c08636533f1af2b417e6e8f460

```

@TheAssassin I'll try with removing the sed statement but ping in case it may be appimaged's fault too.

**Comment by TheLastProject** [comment]
Fixed in c3e9905a7ee4dc75a7ba33858cbddddc5c698e90

**Comment by TheAssassin** [comment]
@TheLastProject that sed was a leftover workaround, and yeah, it's been safe to remove.

appimaged and my tool [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher) (well, any tool that uses libappimage for desktop integration) work like replacing the first part of an `Exec` command with the full path to the AppImage. Use case in which this works:

```
Exec=myapp %f
```

will become

```
Exec=/path/to/myapp.AppImage %f
```
